### ISSUE TITLE
HM Himel Sen

### ISSUE THREAD

**Issue by Himelsen** [DataStax-Academy/Intro-to-Cassandra-for-Developers#297]
**Name:** Himel Sen
**Email:** himelsen@gmail.com
**Linkedin Profile:** https://www.linkedin.com/in/himel-sen-1314ab23
![CassandraSC](https://user-images.githubusercontent.com/88796960/129083678-e217d771-c129-4836-aa0c-3dda01501136.png)

Attach the homework screenshots below for both step II and step III:
-----------------------------------------

<SCREENSHOT>

**Comment by 000** [comment]
You may need to attach "Congratulations!" screenshots from https://www.datastax.com/node/3272 and https://www.datastax.com/dev/scenario/try-it-out-cassandra-query-language-cql as well, @Himelsen .

**Comment by SonicDMG** [comment]
That's correct @000, thank you for pointing that out. Himel, please complete those 2 scenarios and provides the completed screens as Tanat mentioned.
